Bot_project/main.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <string>
+
+using namespace std; //I know using namespace std is bad practice, but i'm just testing stuff out.
+
+//a function that doesn't ask for a name
+//so I can call and return for a loop
+//(so the program stays awake).
+//Definitely not the best way but I'll change it later.
+
+int prompt(const string& input_name )
```

Comment:
What are the possible values returned from this function? What does each value mean?
We need to define that and stick to it... and then the caller should use it according to the definition.